packages/widgets/src/widgets/markdown/Markdown.tsx

```diff
@@ -18,29 +15,28 @@ export interface IProps extends HTMLProps<HTMLDivElement> {
 
 function Markdown (props: IProps, _ref: ForwardedRef<HTMLDivElement>) {
   const { markdown, className, forwardedRef, ...rest } = props;
-  const html = useMarkdown(markdown);
+  const jsx = useMarkdown(markdown);
   return (
-    <div ref={forwardedRef} className={clsx(className, 'markdown-body p-2')} dangerouslySetInnerHTML={{ __html: html }} {...rest} />
+    <div ref={forwardedRef} className={clsx(className, 'markdown-body p-2')} {...rest}>
+      {jsx}
+    </div>
   );
 }
 
 function useMarkdown (markdown) {
-  const [html, setHtml] = useState('');
+  const [Content, setContent] = useState(<></>)
 
   useEffect(() => {
-    const file = unified()
-      .use(remarkParse)
-      .use(remarkRehype, { allowDangerousHtml: true })
-      .use(rehypeRaw)
-      .use(rehypeSanitize)
-      .use(rehypeStringify)
+    unified()
+      .use(rehypeParse, {fragment: true})
```

Comment:
`unified` should use `remarkParse`, `remarkRehype` and `rehypeReact`

- `remarkParse` parses markdown to AST
- `remarkRehype` convert markdown AST to html AST
- `rehypeReact` render html AST to react nodes